haskell/src/Monocle/Api/Server.hs

```diff
@@ -299,6 +299,36 @@ crawlerCommitInfo request = do
         . Enumerated
         $ Right err
 
+-- | /suggestions endpoint
+searchSuggestions :: SearchPB.SuggestionsRequest -> AppM SearchPB.SuggestionsResponse
+searchSuggestions request = do
+  tenants <- getConfig
+  let SearchPB.SuggestionsRequest {..} = request
+
+  let tenantM = Config.lookupTenant tenants (toStrict suggestionsRequestIndex)
+
+  case tenantM of
+    Just tenant -> do
+      now <- getCurrentTime
+      runTenantQueryM tenant (emptyQ now) $ do Q.getSuggestions
+    Nothing ->
+      -- Simply return empty suggestions in case of unknown tenant
+      pure $
+        SearchPB.SuggestionsResponse
+          (V.fromList [])
+          (V.fromList [])
+          (V.fromList [])
+          (V.fromList [])
+          (V.fromList [])
+  where
+    emptyQ now' =
+      let queryGet = Q.blankQueryModifier
+          queryMinBoundsSet = True
+          oneYearAgo date = Q.subUTCTimeSecond date (3600 * 24 * 365)
+          now'' = Q.dropTime now'
+          queryBounds = (oneYearAgo now'', now'')
+       in Q.Query {..}
```

Comment:
It seems like we could use a `blankQuery :: UTCTime {- since date -} -> Query` helper in the Query module along with `yearAgo :: UTCTime -> UTCTime` helper to do something like this here:

```suggestion
    emptyQ = Q.blankQuery . Q.yearAgo
```